src/components/common/Description/GenericDescription.scss

```diff
@@ -0,0 +1,140 @@
+.tab-list {
```

Comment:
this should be wrapped in parent container class

Reply:
done